src/content/docs/synthetics/synthetic-monitoring/getting-started/intro-synthetic-monitoring.mdx

```diff
@@ -0,0 +1,320 @@
+---
```

Comment:
Is this page in the Nav? I can't seem to find it anywhere but it could just be Friday eyes.

Reply:
Hmmmm. Good catch and i probably forgot!

Reply:
Added!